requirements/edx/testing.txt

```diff
@@ -952,7 +952,7 @@ openedx-calc==3.0.1
     # via -r requirements/edx/base.txt
 openedx-django-wiki==1.1.4
     # via -r requirements/edx/base.txt
-openedx-events==3.2.0
+openedx-events==4.1.0
```

Comment:
Hello @rgraber!
This version bump includes [this](https://github.com/openedx/openedx-events/pull/154) breaking change, should we consider something before merging?